CHANGELOG.md

```diff
@@ -4,6 +4,102 @@ All notable changes to Router will be documented in this file.
 
 This project adheres to [Semantic Versioning v2.0.0](https://semver.org/spec/v2.0.0.html).
 
+# [1.26.0] - 2023-07-28
+
+## 🚀 Features
+
+### Add coprocessor metrics ([PR #3483](https://github.com/apollographql/router/pull/3483))
+
+Introduces a new metric for the router:
+
+```
+apollo.router.operations.coprocessor
+```
+
+It has two attributes:
+
+```
+coprocessor.stage: string (RouterRequest, RouterResponse, SubgraphRequest, SubgraphResponse)
+coprocessor.succeeded: bool
+```
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3483
+
+### Constrain APOLLO_ROUTER_LOG and --log global levels to the router ([Issue #3474](https://github.com/apollographql/router/issues/3474))
+
+`APOLLO_ROUTER_LOG` and `--log` now implicitly set a filter constraining the logging to the `apollo_router` module, simplifying the debugging experience for users.
+
+For advanced users `RUST_LOG` can be used for standard log filter behavior.
+
+Thus:
+
+```
+RUST_LOG=apollo_router=warn
+--log warn
+APOLLO_ROUTER_LOG=warn
+```
+
+are equivalent with all three statements resulting in `warn` level logging for the router.
+
+For more details, read the logging configuration documentation.
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3477
+
+### Add support for PodDisruptionBudget to helm chart ([Issue #3345](https://github.com/apollographql/router/issues/3345))
+
+A [PodDisuptionBudget](https://kubernetes.io/docs/tasks/run-application/configure-pdb/) may now be specified for your router to limit the number of concurrent disruptions.
+
+Example Configuration:
+
+```yaml
+podDisruptionBudget:
+  minAvailable: 1
+```
+
+By [@garypen](https://github.com/garypen) in https://github.com/apollographql/router/pull/3469
+
+## 🐛 Fixes
+
+### Don't hide `--dev` from `--help` ([Issue #2705](https://github.com/apollographql/router/issues/2705))
+
+Display documentation about `--dev` when launching the router with `--help` argument.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/3479
+
+### Fix default rhai script dir for Windows ([Issue #3401](https://github.com/apollographql/router/issues/3401))
+
+Using default `rhai.scripts` field won't end up in an error.
+
+By [@bnjjj](https://github.com/bnjjj) in https://github.com/apollographql/router/pull/3411
+
+### Fix the prometheus descriptions as well as the metrics ([Issue #3491](https://github.com/apollographql/router/issues/3491))
+
+I didn't realise the descriptions on the prometheus stats were significant, so my prefious prometheus fix constrained itself to renaming the actual metrics.
+
```

Comment:
I just spotted a typo: `prefious` -> `previous`

Reply:
fixed in 9c67c324